### DISCUSSION TITLE
Can't find access log

### DISCUSSION THREAD

**Discussion by yannrenarddav** [projectcontour/contour#5989] [Q&A] [answered]
Hi,

We have an application complaining about lost request. I can see the request on the external load balancer, now I would like to check on contour/envoy if everything is OK. Reading the documentation I can see that it's native but I don't see anything on logs, or I don't look at the right place. Can you help me to find the logs please ? 

$ kubectl get httpproxy -A
NAMESPACE   NAME    FQDN                     TLS SECRET   STATUS   STATUS DESCRIPTION
test        hello   test.k8s-stg.c2.dav.fr                valid    Valid HTTPProxy 

-> I would like to trace connections to that (it's a test cluster but I have exacly the same thing in production with a real app). 

$ kubectl logs -l app=envoy -n projectcontour
*Defaulted container "shutdown-manager" out of: shutdown-manager, envoy, envoy-initconfig (init)
Defaulted container "shutdown-manager" out of: shutdown-manager, envoy, envoy-initconfig (init)
Defaulted container "shutdown-manager" out of: shutdown-manager, envoy, envoy-initconfig (init)
time="2023-11-27T16:36:17Z" level=info msg="started envoy shutdown manager" context=shutdown-manager
time="2023-11-27T16:36:17Z" level=info msg="started envoy shutdown manager" context=shutdown-manager
time="2023-11-27T16:36:27Z" level=info msg="started envoy shutdown manager" context=shutdown-manager

I'm using k8s 1.24.14 and contour v1.23.6 with the default configuration (https://raw.githubusercontent.com/projectcontour/contour/release-1.23/examples/render/contour.yaml), I just changed that : 
$ sdiff -s contour-v1.23.6.yml contour-v1.23.6.yml.ori
    enableExternalNameService: true                           <
        image: harbor.dassault-avion.fr/docker-hub/bitnami/co |         image: ghcr.io/projectcontour/contour:v1.23.6
    nodePort: 80                                              <
    nodePort: 31143                                           <
    nodePort: 30335                                           <
  type: NodePort                                              |   type: LoadBalancer
        image: harbor.dassault-avion.fr/docker-hub/bitnami/co |         image: ghcr.io/projectcontour/contour:v1.23.6
        image: harbor.dassault-avion.fr/docker-hub/bitnami/co |         image: ghcr.io/projectcontour/contour:v1.23.6
#        livenessProbe:                                       |         livenessProbe:
#          httpGet:                                           |           httpGet:
#            path: /healthz                                   |             path: /healthz
#            port: 8090                                       |             port: 8090
#          initialDelaySeconds: 3                             |           initialDelaySeconds: 3
#          periodSeconds: 10                                  |           periodSeconds: 10
        image: harbor.dassault-avion.fr/docker-hub/bitnami/en |         image: docker.io/envoyproxy/envoy:v1.24.10
        image: harbor.dassault-avion.fr/docker-hub/bitnami/co |         image: ghcr.io/projectcontour/contour:v1.23.6

Regards.

**Comment by tsaarni** [comment, marked as answer]
The access logs are in envoy container of the envoy pod. Did you try `kubectl logs -l app=envoy -n projectcontour -c envoy`?

**Comment by yannrenarddav** [comment]
No I didn't try that, thank you it's better ! 👍
